cmd/apps/nginx_app.go

```diff
@@ -84,36 +85,23 @@ flag and the nginx-ingress docs for more info`,
 		}
 
 		chartPath := path.Join(os.TempDir(), "charts")
-		err = fetchChart(chartPath, "stable/nginx-ingress", defaultVersion, helm3)
+		err = fetchChart(chartPath, "ingress-nginx/ingress-nginx", defaultVersion, helm3)
 
 		if err != nil {
 			return err
 		}
 
 		overrides := map[string]string{}
 
-		overrides["defaultBackend.enabled"] = "false"
-
-		arch := getNodeArchitecture()
```

Comment:
This is not required anymore since the release [0.32.0](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-0.32.0)

Reply:
👍